### ISSUE TITLE
[Feature Request] Support Public Class Fields

### ISSUE THREAD

**Issue by aaclayton** [javascript-obfuscator/javascript-obfuscator#822]
**Is your feature request related to a problem? Please describe.**
Public Class Fields are now supported by modern browsers and by Node.js 12+
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields

These field declarations are not currently supported by javascript obfuscator. Examples include:

```js
class ClassWithPublicFields {
  instanceField = "foo";
  static staticField = "bar";
}
```

Both `instanceField` and `staticField` will break the obfuscation process with an "Unexpected token " error.

**Describe the solution you'd like**
I would like for javascript-obfuscator to support these class fields and preserve their functionality in the obfuscated output.

**Describe alternatives you've considered**
Do not use these class fields in code which needs to be obfuscated.

**Additional context**
Please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields for details and current browser/node compatibility.

I found this issue #510 which seems to be related, but was (perhaps incorrectly?) marked as closed. If this issue is blocked by an upstream dependency, it would be great to leave the issue open so it can continue to be tracked until such a time as the upstream blocker is removed. Thank you!

**Comment by sanex3339** [comment]
Duplicate of https://github.com/javascript-obfuscator/javascript-obfuscator/issues/510

**Comment by aaclayton** [comment]
I understand that this is a duplicate, but if this is closed I would petition to reopen #510 so that it can be found as an active issue. If #510 is left open you will receive fewer new issues that are duplicates created.

**Comment by sanex3339** [comment]
510 has been opened all this time
